pkg/reconciler/events/storage/storage.go

```diff
@@ -82,22 +81,22 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, storage *v1alpha1.CloudS
 	ctx = logging.WithLogger(ctx, r.Logger.With(zap.Any("storage", storage)))
 
 	storage.Status.InitializeConditions()
-	storage.Status.ObservedGeneration = storage.Generation
```

Comment:
as with other review. I think this is correct and we should add it back

Reply:
see comment below.